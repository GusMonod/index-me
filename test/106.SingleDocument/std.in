this is an example of a single document (one line)
