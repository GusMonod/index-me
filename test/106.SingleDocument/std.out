this: 0
is: 0
an: 0
example: 0
of: 0
a: 0
single: 0
document: 0
(one: 0
line): 0